src/core/qgis.h

```diff
@@ -3889,6 +3889,27 @@ class CORE_EXPORT Qgis
     Q_DECLARE_FLAGS( TiledSceneRendererFlags, TiledSceneRendererFlag )
     Q_FLAG( TiledSceneRendererFlags )
 
+    /**
+     * Resampling algorithm to be used (equivalent to GDAL's enum GDALResampleAlg)
+     * \note RA_Max, RA_Min, RA_Median, RA_Q1 and RA_Q3 are available on GDAL >= 2.0 builds only
+     * \since QGIS 3.34
+     */
+    enum class GdalResampleAlgorithm : int
```

Comment:
```suggestion
    enum class GdalResampleAlgorithm SIP_MONKEYPATCH_SCOPEENUM_UNNEST( QgsAlignRaster, ResampleAlg ) : int 
```

Reply:
Fixed.

Reply:
Ah, good point.

Yes it'll need to be a manual insertion into that file. But I'd use the (broken) auto generated code from the monkey patch annotation as the starting point template, as e.g. we'll need to monkey patch each of the individual enum values as well as the enum itself (like the monkey patch annotation does)

Reply:
Oh wait, I see you've already done this :+1: